src/components/Navigation/TopBar/Help/Feedback/Feedback.const.ts

```diff
@@ -0,0 +1,24 @@
+import { FeedbackLocalStorage, StatusEnum } from './Feedback.types';
+
+export const MAX_FILE_SIZE_BYTES = 1048575;
+
+export const SERVICE_NOW_QUERY_KEY = 'serviceNowIncident';
+export const SLACK_POST_QUERY_KEY = 'slackPostMessage';
+export const SLACK_FILE_QUERY_KEY = 'slackFileUpload';
+
+export const DEFAULT_REQUEST_ERROR_MESSAGE =
+  'There was an error with the request';
+
+export const ONE_HOUR_IN_MS: number = 3600000;
```

Comment:
Nitpicky but when writing MS constants IMO it's better do just have the full calculation there: `1000 * 60 * 60`